src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -116,20 +112,33 @@ def run_offline_reintegration_test(self, num_pool, data=False):
             self.pool.display_pool_daos_space("Pool space: Beginning")
             pver_begin = self.get_pool_version()
             self.log.info("Pool Version at the beginning %s", pver_begin)
-            output = self.dmg_command.pool_exclude(self.pool.uuid,
-                                                   rank)
+            if server_boot is False:
+                output = self.dmg_command.pool_exclude(self.pool.uuid,
+                                                       rank)
+            else:
+                output = self.dmg_command.system_stop(ranks=rank)
+                time.sleep(10)
```

Comment:
We should refrain from using sleeps in the test code. We can check the system steam status using system query here and avoid sleep.

Reply:
Agree. Will do... Need to check what does system query provide... Good suggestion.

Reply:
Removed. Done.

Reply:
If self.pool.wait_for_rebuild() is used, it will take care of DER_UNREACH

Reply:
Great. Will try it...